app/src/controllers/review/services/ReviewService.ts

```diff
@@ -7,11 +7,13 @@ import { AuthorizationError, NotFoundError } from '@errors/ServiceErrors'
 import ReviewRepository from '@review/repositories/ReviewRepository'
 import ShopRepository from '@review/repositories/ShopRepository'
 import UserRepository from '@review/repositories/UserRepository'
+import { ReviewScore } from '@request-response-types/models/Review'
```

Comment:
you see how there's nothing from the request-response-types inside the service, it's supposed to be in the controller only